src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  let inverse = {};
+
+  for (const key in robot) {
+    if (inverse[robot[key]]) {
+      inverse = null;
+      break;
```

Comment:
```suggestion
      return null;
```
We can immediately `return` - it's more transparent, and shorter, no need `let` variable